pkg/graph/schema.graphql

```diff
@@ -1552,6 +1553,83 @@ input UpdateTRBRequestAttendeeInput {
   role: PersonRole!
 }
 
+"""
+Represents a lifecycle cost phase
+"""
+enum TRBWhereInProcessOption {
+  I_HAVE_AN_IDEA_AND_WANT_TO_BRAINSTORM
+  CONTRACTING_WORK_HAS_STARTED
+  DEVELOPMENT_HAS_RECENTLY_STARTED
+  DEVELOPMENT_IS_SIGNIFICANTLY_UNDERWAY
+  THE_SYSTEM_IS_IN_OPERATION_AND_MAINTENANCE
+  OTHER
+  UNKNOWN
```

Comment:
When is this `UNKNOWN` value used? I'm not sure what it'd correspond to in the model/database enum type; if it's meant to represent an unfilled field on the form, that model's field shouldn't be a pointer (so it's non-nullable), as should the `where_in_process` column in the database table.

Reply:
Ah, this was removed from the DB schema but never from the GQL schema, we should probably clean this up here!

Reply:
@jstrang-oddball I see this was marked as resolved but it looks like the option is still in the schema - just wanted to make sure this wasn't missed (unresolving for now)